addon/-private/class/modifier.ts

```diff
@@ -58,25 +62,33 @@ export default class ClassBasedModifier<
    * Called when the modifier is installed **and** anytime the arguments are
    * updated.
    */
-  didReceiveArguments() {}
+  // Disabled because this exists to be a no-op (as well as for docs).
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  didReceiveArguments(): void {}
```

Comment:
I usually put a comment in the function body explaining why it’s empty to avoid having to explicitly disable this rule

Reply:
Ahhh, that's a great idea. Will do that!